juntagrico/templates/messages/error.html

```diff
@@ -0,0 +1,5 @@
+{% load i18n %}
+{% load config %}
+<div class="alert alert-danger">
+    {% trans "Oops! Da ist leider etwas schiefgegangen. Bitte versuche es noch einmal." %}
```

Comment:
to generic, retry won't help if the job is fully booked, should be fully_booked_error.html or similar with a specific message like "Leider hat es in der zwischenzeit keinen platz mehr für die von dir gewählte Anzal Teilnehmer" or so

Reply:
The idea was to have a generic error for different cases here and elsewhere after a POST submission. Normally a user should not get there unless he keeps a job page open in his browser. And he will not get the error again, as the option to subscribe will disappear on reload. To split up, we would need:

- fully booked error
- not enough slots error
- job is in the past error
- job is running error
- ...

Isn‘t this too much?

Reply:
ok maybe you are right but in all the cases mentioned retry would not help so this part of the text should be removed
and the text should be like it is not possible to either participate or participate in this configuration

Reply:
Together with this error, the options for a possible retry are already updated on the page, so a retry (if possible) will not result in another error. The idea for this generic error was to be able to use it in different cases here and elsewhere after a POST submission. If you want the text to be specific to this error, how do you want me to formulate it?